docs/reference/schemas/schema-uris.md

```diff
@@ -208,9 +208,9 @@ The following list of tables defines the recognized URIs for the configuration d
   | Canonically bundled     | `v3`     | `https://aka.ms/dsc/schemas/v3/bundled/config/document.json`            |
   | Canonically bundled     | `v3.0`   | `https://aka.ms/dsc/schemas/v3.0/bundled/config/document.json`          |
   | Canonically bundled     | `v3.0.0` | `https://aka.ms/dsc/schemas/v3.0.0/bundled/config/document.json`        |
-  | Enhanced AUthoring      | `v3`     | `https://aka.ms/dsc/schemas/v3/bundled/config/document.vscode.json`     |
-  | Enhanced AUthoring      | `v3.0`   | `https://aka.ms/dsc/schemas/v3.0/bundled/config/document.vscode.json`   |
-  | Enhanced AUthoring      | `v3.0.0` | `https://aka.ms/dsc/schemas/v3.0.0/bundled/config/document.vscode.json` |
+  | Enhanced Authoring      | `v3`     | `https://aka.ms/dsc/schemas/v3/bundled/config/document.vscode.json`     |
+  | Enhanced Authoring      | `v3.0`   | `https://aka.ms/dsc/schemas/v3.0/bundled/config/document.vscode.json`   |
+  | Enhanced Authoring      | `v3.0.0` | `https://aka.ms/dsc/schemas/v3.0.0/bundled/config/document.vscode.json` |
```

Comment:
We should consistently use sentence casing for these values, as with the other entries.

```suggestion
  | Enhanced authoring      | `v3`     | `https://aka.ms/dsc/schemas/v3/bundled/config/document.vscode.json`     |
  | Enhanced authoring      | `v3.0`   | `https://aka.ms/dsc/schemas/v3.0/bundled/config/document.vscode.json`   |
  | Enhanced authoring      | `v3.0.0` | `https://aka.ms/dsc/schemas/v3.0.0/bundled/config/document.vscode.json` |
```